src/Octoshift/Extensions/StringExtensions.cs

```diff
@@ -26,5 +26,29 @@ public static class StringExtensions
         public static string EscapeDataString(this string value) => Uri.EscapeDataString(value);
 
         public static byte[] ToBytes(this string s) => Encoding.UTF8.GetBytes(s);
+
+        public static bool IsUrl(this string s)
```

Comment:
Would prefer we use standard dotnet core library to validate something like below referencing https://learn.microsoft.com/en-us/dotnet/api/system.uri?view=net-8.0 
```csharp
public static bool IsUrl(this string s)
{
    if (s.IsNullOrWhiteSpace())
    {
        return false;
    }

    return Uri.TryCreate(s, UriKind.Absolute, out var uri) 
        && (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps);
}
```